playwright/tests/pageobjects/InscriptionView.ts

```diff
@@ -46,7 +46,7 @@ export class InscriptionView {
     await InscriptionView.hideQueryDevTools(page);
   }
 
-  accordion(partName: string) {
+  accordion(partName: string): Accordion {

```

Comment:
why needed?

Reply:
no need; just thought we would get better type support if strictly defined?

Reply:
not really. typescript infers the return type from the return value

Reply:
This is only helpful if you want to ensure that every return will return the type you define